docs/GettingStarted.md

```diff
@@ -267,7 +267,7 @@ the ``Pin`` object attached to the client. Example:
     end
 
     client = Net::HTTP.new(host, port)
-    Datadog::Pin.get_from(client).config = { distributed_tracing: true }
+    Datadog.configure(client).config = { distributed_tracing: true }
```

Comment:
This is clearly not ideal. Perhaps we should alias `Pin#config` to `Pin#extra` so we can have something like:

```rb
client = Net::HTTP.new(host, port)

Datadog.configure(client) do |c|
  c.service_name = 'custom-service'
  c.extra = { distributed_tracing: true }
end
```

The upside of creating just the alias is that we're still using the `Pin` object underneath and we're not making any breaking changes. On the other hand, it might be worth it to create a decorator around `Pin` that better conforms to the `Datadog.configure` API instead of just yielding the `Pin` object as it is now.

Reply:
Created a PR improving that https://github.com/DataDog/dd-trace-rb/pull/299